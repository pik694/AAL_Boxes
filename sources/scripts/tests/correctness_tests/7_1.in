7
58 100 89
83 76 93
86 26 86
83 44 32
38 57 18
61 17 84
54 85 35

