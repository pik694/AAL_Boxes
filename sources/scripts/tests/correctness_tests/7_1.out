4 3 6 1 5 2 0 
1698750->1102844
